pkg/model/api/v1/variable.go

```diff
@@ -84,6 +85,8 @@ func (v *GlobalVariable) GetSpec() interface{} {
 	return v.Spec
 }
 
+// Variable is a global variable at project level.
+// If you are looking for the variable contained in a dashboard, take a look at dashboard.Variable
```

Comment:
Variable relates to variables defined at project level.
If you are looking for variable defined at dashboard level, see dashboard.Variable